### DISCUSSION TITLE
Upgrade to 1.4.5 possible?

### DISCUSSION THREAD

**Discussion by acer8888** [Nico640/docker-unms#72] [Q&A] [answered]
@Nico640 hey love what you've done with unms (UISP) to docker it. I'm curious if you have time to update it to 1.4.5? I would do it myself but I'm a docker newb. I'm trying to migrate from Ubiquiti's hosted UISP, but they are on 1.4.5 and I cannot import into the docker version because it's 1.4.4, on a little bit of a time crunch due to not having enough devices in order to keep Ubiquiti hosting. I understand tho if you don't have time. Thanks you for the awesome container!

**Comment by Nico640** [comment, marked as answer]
Hi, I just pushed the tag `1.4.5-beta`, which you should be able to use instead of the `latest` tag. I didn't expect the UISP Cloud to be on 1.4.5, because it never went public, 1.4.4 is still the latest stable release. Just be sure to switch back to the latest tag when it has been updated with a future stable version.

**Reply by acer8888** [reply]
Your awesome! Thank you for doing that. :)
